values-tls.yaml

```diff
@@ -0,0 +1,172 @@
+# Default values for infinispan-helm-charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+images:
+  # [USER] The container images for server pods.
+  server: quay.io/infinispan/server:14.0
+  initContainer: registry.access.redhat.com/ubi8-micro
+
+deploy:
+  # [USER] Specify the number of nodes in the cluster.
+  replicas: 1
+
+  clusterDomain: cluster.local
```

Comment:
What's this field for?

Reply:
it was already there: it's a part of the [dns ping query](https://github.com/infinispan/infinispan-helm-charts/blob/4ea140dbcc9ec2a9511f838eaced6e3aff8c982b/templates/statefulset.yaml#L80)

Reply:
Argh yes, it's been a while since I worked on the templates :sweat_smile: